ls1intum/Artemis
# Fix to not shown Instructor Buttons

### Checklist

### Motivation and Context


Currently instructor buttons are shown to let the instructor assess submissions even if the student did not submit the exam. The instructor now has to set the exam to submitted before being able to da that. This makes the behavior more consistent

### Description

We add additional checks to see if the exam was submitted or not. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Exam -> Student Exam -> choose one that was not submitted in time
3. See if the buttons ONLY show up, once you set the exam to submitted. 

### Test Coverage



student-exam-detail-table-row.component.ts: 75%
student-exam-detail.component.ts: 86.75
### Screenshots
